app/javascript/controllers/search_controller.js

```diff
@@ -0,0 +1,22 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["submit"]
+
+  connect() {
+    this.timeout = null
+  }
+
+  submit() {
+    clearTimeout(this.timeout)
+    this.timeout = setTimeout(() => {
+      this.submitForm()
+    }, 200)
+  }
+
+  // NOTE: form.submit() doesn't seem to play well with Turbo at the moment -
+  // so just click the hidden submit button
+  submitForm() {
+    this.submitTarget.click()
```

Comment:
I double checked ... and indeed calling `this.element.submit()` on the `<form>` still doesn't work with Turbo.  It submits a non-turbo-stream-`Accept` request.  Which isn't what I want.